frontend/src/sdk/createStore.ts

```diff
@@ -1,106 +1,25 @@
 import { getActivePlugins } from '../Utils/plugins';
 import packageInfo from '../../package.json';
-import { PluginLoader, PluginLoaderOptions, PluginStore } from '@openshift/dynamic-plugin-sdk';
-import type { To, NavigateOptions } from 'react-router-dom';
+import { PluginLoader, PluginStore } from '@openshift/dynamic-plugin-sdk';
 
-const calculateTo = (to: To) => {
-  if (typeof to === 'string' && !to.startsWith('/hac') && to.startsWith('/')) {
-    return `/hac${to}`;
-  } else if (typeof to !== 'string' && to.pathname && !to.pathname.startsWith('/hac')) {
-    return {
-      ...to,
-      pathname: `/hac${to.pathname}`,
-    };
+export const createStore = (store?: PluginStore) => {
+  let internalStore = store;
+  if (!internalStore) {
+    const loader = new PluginLoader({
+      postProcessManifest: (manifest) => ({
+        ...manifest,
+        loadScripts: manifest.loadScripts ?? ['plugin-entry.js'],
+        registrationMethod: manifest.registrationMethod ?? 'callback',
+      }),
+    });
+    internalStore = new PluginStore();
+    internalStore.setLoader(loader);
```

Comment:
Assuming you aim to support loading plugins with `callback` registration method, don't forget to call

```ts
loader.registerPluginEntryCallback();
```

when creating a new `PluginStore` instance.